### PR TITLE
Tweaking some things

### PR DESCRIPTION
**Pull request by moxiegirl**
Signed-off-by: Mary Anthony mary@docker.com

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -26,9 +26,7 @@ contains the last release of documentation.
 
 After a release, documentation updates are continually merged into `master` as
 they occur. This work includes new documentation for forthcoming features, bug
-fixes, and other updates. Docker's CI system automatically builds and updates
-the `master` documentation after each merge and posts it to
-[http://docs.master.dockerproject.org](http://docs.master.dockerproject.org). 
+fixes, and other updates.
```
Comment (thaJeztah) [inline docs/README.md]: Just wondering; Will this return in some form? I actually liked being able to point users to that to view upcoming docs changes

File: docs/README.md [hunk 2]
```diff
@@ -67,17 +65,16 @@ own.
 	The `docker/docker` repository contains a `Dockerfile` and a `Makefile`.
 	Together, these create a development environment in which you can build and
 	run a container running the Docker documentation website. To build the
-	documentation site, enter `make docs` at the root of your `docker/docker`
-	fork:
-	
+	documentation site, enter `make docs` in the `docs` directory of your `docker/docker` fork:
```
Comment (thaJeztah) [inline docs/README.md]: Oh? This changed? `make docs` from the root no longer works?

Comment (moxiegirl) [inline docs/README.md]: You have to be in docs to `make docs` --- some projects added this to root Makefile not all of them. So yes, this is the universal way to do it.

Comment (thaJeztah) [inline docs/README.md]: Ah, thanks, yup that makes sense then (wasn't aware of the other repos being different there)

**Comment by thaJeztah** [comment]
LGTM, thanks!

**Comment by thaJeztah** [comment]
Do you want this cherry picked?

**Comment by moxiegirl** [comment]
@thaJeztah Nah...no need for that...not part of the docs publish. Thanks for asking tho.

**Pull request merged**